ninetofiver/urls.py

```diff
@@ -52,6 +52,7 @@
 router.register(r'my_performances', views.MyPerformanceViewSet, base_name='myperformance')
 router.register(r'my_attachments', views.MyAttachmentViewSet, base_name='myattachment')
 router.register(r'my_workschedules', views.MyWorkScheduleViewSet, base_name='myworkschedule')
+router.register(r'redmine_time_entries', views.RedmineTimeEntryViewSet, base_name='redmineTimeEntry')
```

Comment:
Why are you camelcasing the basename of the route here? We don't do this anywhere else.